Delcam.ProductInterface.PowerMILL.Test/PMEntityTests/PMModelTest.cs

```diff
@@ -173,6 +173,22 @@ public void MoveTest()
             Assert.AreEqual(bxAfter.MaximumBounds - bxBefore.MaximumBounds, moveBy);
         }
 
+        [Test]
+        public void RotateTest()
+        {
+            // #1
+            // Rotate by 90° in X-Axis
+            _defaultModel.Rotate(Axes.X, 90);
+
+            // #2
+            // Rotate by 90° in Y-Axis
+            _defaultModel.Rotate(Axes.Y, 90);
+
+            // #3
+            // Rotate by 90° in Z-Axis
+            _defaultModel.Rotate(Axes.Z, 90);
+        }
```

Comment:
In order for this to be a "Test" you need to use an Assert test to see if what you wanted to do actually happened.  I would suggest checking something like the bounding box after rotation.  Take a look at the MoveTest for example.

Reply:
OK, i've changed the "Test" and compare the bounding box after rotation.